### PR TITLE
[HAL-1079] Adding expand all button to show all tree in deployment detail

### PR DESCRIPTION
**Pull request by khermano**
Issue: https://issues.redhat.com/browse/HAL-1079

### CODE REVIEW

**Comment by hpehl** [comment]
Thanks, @khermano, for your contribution. I plan another HAL release for Thursday and will review your PR by then.

File: core/src/main/java/org/jboss/hal/core/modelbrowser/ModelBrowser.java [hunk 1]
```diff
@@ -169,6 +170,10 @@ public ModelBrowser(CrudOperations crud,
                         .on(click, event -> refresh(tree.getSelected()))
                         .title(resources.constants().refresh())
                         .add(Elements.i().css(fontAwesome(CSS.refresh))).element())
+                .add(expandAll = button().css(btn, btnDefault)
```
Comment (hpehl) [inline core/src/main/java/org/jboss/hal/core/modelbrowser/ModelBrowser.java:173]: Please remove the expand all feature from the model browser. The tree is too large. The page crashes, when I click on expand all on my machine.

**Review by hpehl — CHANGES_REQUESTED** [review]

File: core/src/main/java/org/jboss/hal/core/modelbrowser/ModelBrowser.java (hunk 1 above)
Comment (khermano) [reply core/src/main/java/org/jboss/hal/core/modelbrowser/ModelBrowser.java]: I am sorry about that. I removed it.

Comment (hpehl) [reply core/src/main/java/org/jboss/hal/core/modelbrowser/ModelBrowser.java]: No worries. Besides that: Good job!
Thanks for your contribution @khermano

**Review by hpehl — APPROVED** [review]

File: core/src/main/java/org/jboss/hal/core/modelbrowser/ModelBrowser.java (hunk 1 above)
Comment (khermano) [reply core/src/main/java/org/jboss/hal/core/modelbrowser/ModelBrowser.java]: Thank you. :)

**Pull request merged**